include/tvm/relax/transform.h

```diff
@@ -223,6 +223,8 @@ TVM_DLL Pass FuseOps(int fuse_opt_level = -1);
  * of a fused function after successful matching.
  * \param patterns The patterns to detect. The order of the patterns determines the order
  * of priority in which they are matched. Higher-priority patterns should come earlier in the list.
+ * \param checks The callback functions that takes a match result and returns a boolean value to
```

Comment:
Taking a match result would work for simple cases, but will it still work for more complicated ones like MHA? I think it would be difficult to retrieve useful information from the match result alone, if the expr contains multiple nodes with similar nature. OTOH, It seems that only the call expr and the match result are available at the call site of checks function.